src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieType } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+
+export type MovieList = {
+  movies?: MovieType[]
+};
+
+export const MoviesList: React.FC<MovieList> = ({ movies = [] }) => (
```

Comment:
```suggestion
export const MoviesList: React.FC<MovieList> = ({ movies }) => (
```
in that way you dont need it here